ui-tests/src/test/java/org/openmrs/reference/AddUserTest.java

```diff
@@ -55,7 +62,8 @@ public void addUserTest() throws Exception {
             assertTrue(manageUserPage.getUserSavedNotification().contains("User Saved"));
 
             homePage = new HomePage(goToLoginPage().login("super_nurse", "Nurse123"));
-            homePage.getLoggedUsername().contains("super_nurse");
+            homePage.waitForPage();
```

Comment:
The home page takes a bit longer to load after login, waiting for page resolves the issue